deepmd/main.py

```diff
@@ -751,6 +751,26 @@ def main_parser() -> argparse.ArgumentParser:
     )
     parser_convert_backend.add_argument("INPUT", help="The input model file.")
     parser_convert_backend.add_argument("OUTPUT", help="The output model file.")
+
+    # * show model ******************************************************************
+    parser_show = subparsers.add_parser(
+        "show",
+        parents=[parser_log],
+        help="(Supported backend: PyTorch) Show the information of a model",
+        formatter_class=RawTextArgumentDefaultsHelpFormatter,
+        epilog=textwrap.dedent(
+            """\
+        examples:
+            dp show model.pt --list-model-branch
+        """
+        ),
+    )
+    parser_show.add_argument("INPUT", help="The input model file")
```

Comment:
It looks to me like only the checkpoint file is supported. The frozen model file is not supported.

Reply:
I have supported both checkpoint file and frozen model file.